wicket-extensions/src/main/java/org/apache/wicket/extensions/ajax/markup/html/modal/ModalDialog-skin.css

```diff
@@ -0,0 +1,24 @@
+body.modal-dialog-open {
+	
+}
+
+body.modal-dialog-no-scroll {
+}
+
+.modal-dialog-overlay {
+	background: rgba(0, 0, 0, 0.2);
+}
+
+.modal-dialog {
+	max-width: 800px;
+	background: white;
+	box-shadow: 0 0 60px 10px rgba(0, 0, 0, 0.7);
```

Comment:
Just my opinion: I'd say 60px is too strong for default style. Maybe 0 0 20px 0 rgba(0,0,0,0.7); ?